pallets/auction/src/lib.rs

```diff
@@ -1080,6 +1096,72 @@ pub mod pallet {
 					<ItemsInAuction<T>>::insert(item_id, true);
 					Ok(auction_id)
 				}
+				ItemId::StackableNFT(class_id, token_id, amount) => {
+					// Check available balance
+					//let available_balance = T::NFTHandler::get_stackable_nft_balance(&recipient, &(class_id,
```

Comment:
If it's not usable, we should remove the commented out code

Reply:
just removed it. It was left from the old implementation